cob_base_drive_chain/ros/src/cob_base_drive_chain.cpp

```diff
@@ -430,7 +430,7 @@ class NodeClass
 		bool srvCallback_Init(cob_srvs::Trigger::Request &req,
 							  cob_srvs::Trigger::Response &res )
 		{
-			ROS_DEBUG("Service Callback init");
+			ROS_DEBUG("Service callback /base_controller/init");
```

Comment:
@ipa-mig-tk @ipa-mig 
even if this is only ROS_DEBUG, can we get rid of the hardcoded namespace ```/base_controller``` in the text - as this is going to change anyway.